### PR TITLE
[pylint] Fix logging-not-lazy

### PR DESCRIPTION
**Pull request by pponnuvel**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [ ] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

File: sos/utilities.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 use for detecting binary files. A less effective method will be used. It is \
 recommended to install proper python3-magic package with the module.
 """
-    log.warning('\n' + fill(msg, 72, replace_whitespace=False) + '\n')
+    log.warning('\n%s\n', fill(msg, 72, replace_whitespace=False))
```
Comment (arif-ali) [inline sos/utilities.py:46]: could you put the whole thing in an f-string here? would that work?

Comment (pponnuvel) [reply sos/utilities.py]: Yes, that should work too. Modified. Thanks.

File: sos/report/__init__.py [hunk 2]
```diff
@@ -1794,8 +1794,8 @@ def compile_tags(ent, key='filepath'):
 
     def _merge_preset_options(self):
         # Log command line options
-        msg = "[%s:%s] executing 'sos %s'"
-        self.soslog.info(msg % (__name__, "setup", " ".join(self.cmdline)))
+        self.soslog.info("[{__name__}:setup] executing "
```
Comment (arif-ali) [inline sos/report/__init__.py:1797]: missed the f for f-string

File: sos/report/__init__.py [hunk 3]
```diff
@@ -1794,8 +1794,8 @@ def compile_tags(ent, key='filepath'):
...
+                         "'sos {' '.join(self.cmdline)}'")
```
Comment (arif-ali) [inline sos/report/__init__.py:1798]: and here

**Review by arif-ali — COMMENTED** [review]
forgot these ones in the last review

File: sos/report/__init__.py (hunk 2 above)
Comment (pponnuvel) [reply sos/report/__init__.py]: Fixed.

File: sos/report/__init__.py (hunk 3 above)
Comment (pponnuvel) [reply sos/report/__init__.py]: Fixed.

**Review by arif-ali — APPROVED** [review]

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**